sdk/spring/azure-spring-boot-starter-keyvault-certificates/src/main/java/com/azure/spring/security/keyvault/certificates/starter/KeyVaultCertificatesEnvironmentPostProcessor.java

```diff
@@ -27,11 +27,6 @@
     @Override
     public void postProcessEnvironment(ConfigurableEnvironment environment, SpringApplication application) {
 
-        if (environment.getProperty("azure.keyvault.uri") == null) {
-            return;
-        }
-
-        putEnvironmentPropertyToSystemProperty(environment, "azure.keyvault.aad-authentication-url");
```

Comment:
I'd suggest to add a warning message in log to warn users still having this property configed

Reply:
Maybe it's not necessary,  because:

1. If customer use `azure-spring-boot-starter-keyvault-certificates`, `azure.keyvault.aad-authentication-url` never exist in `KeyVaultProperties.java`, so customer never know this property exists.

Refs: https://github.com/Azure/azure-sdk-for-java/commits/7989020420f8354b9f6c1597d68c06acddab63d8/sdk/spring/azure-spring-boot-starter-keyvault-certificates/src/main/java/com/azure/spring/security/keyvault/certificates/starter/KeyVaultProperties.java


2. If customer use `azure-security-keyvault-jca`, this already written in changelog:

Refs: https://github.com/Azure/azure-sdk-for-java/blob/7989020420f8354b9f6c1597d68c06acddab63d8/sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md#breaking-changes